test/mongo.test.js

```diff
@@ -1,24 +1,23 @@
 /*
   MIT License,
-  Copyright (c) 2010-2020, Richard Rodger and other contributors.
+  Copyright (c) 2010-2021, Richard Rodger and other contributors.
 */
 
 'use strict'
 
 var Util = require('util')
-var Assert = require('assert')
+const Assert = require('assert')
 
 var Seneca = require('seneca')
 var Async = require('async')
 
 const Lab = require('@hapi/lab')
 const Code = require('@hapi/code')
-const lab = (exports.lab = Lab.script())
 const expect = Code.expect
 
-const describe = lab.describe
+const lab = (exports.lab = Lab.script())
```

Comment:
Only do this if you intend to make this change in the other repos that use Lab

Reply:
If `exports.lab` is removed, I am afraid this happens:

> $ npm run test
> The file: seneca-mongo-store/test/mongo.test.js includes a lab script that is not exported via exports.lab